lms/djangoapps/open_ended_grading/views.py

```diff
@@ -109,6 +114,8 @@ class OpenEndedGradingTab(EnrolledTab):
 
     @classmethod
     def is_enabled(cls, course, user=None):  # pylint: disable=unused-argument
+        if XBlockDisableConfig.is_block_type_disabled('combinedopenended'):
+            return False
```

Comment:
@dianakhuang I have updated the `is_enabled` methods of the tabs. Can you review again?